style.css

```diff
@@ -341,3 +369,124 @@ textarea {
   color: #42526e;
   line-height: 32px;
 }
+
+@media screen and (min-width: 750px) {
```

Comment:
* This min-width value should serve minimum width for the desktop version and better to make it in the range 992px-1100px. Yours is a bit smaller than standard desktop widths.

Reply:
- This issue is well addressed! :clap: